libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/UrlUtils.java

```diff
@@ -49,6 +49,10 @@ public static String getHost(final String urlString) {
         return "";
     }
 
+    public static boolean checkIfContentUri(String uri) {
```

Comment:
I feel like `checkIfContentUri` name is a bit ambiguous.  Maybe we could rename it to something like `isContentUri`? WDYT?

Reply:
Nice. Thanks!